README.md

```diff
@@ -4,282 +4,49 @@
+## Installation and CLI Help
 ```
-npx @nrfcloud/device-simulator-v2 -k <api key> [-d <desired device ID>] -a -s gps,acc,device,temp
-```
-
-## Options
-These are the options. Most of them are set with environment variables.
-
-```
-  -k, --api-key <apiKey> (required)                  API key for nRF Cloud (default: "")
-  -h, --api-host <apiHost>                           API host for nRF Cloud (default: "https://api.nrfcloud.com")
-  -d, --device-id <deviceId>                         ID of the device (default: <nrfsim-randomString>)
-  -a, --onboard <onboardingType>                     Onboard the device with "jitp", or "preconnect"
-  -s, --services <services>                          Comma-delimited list of services to enable. Any of: [gps,acc,temp,device,rsrp,gnss,location,log,alert]
-  -f, --app-fw-version <appFwVersion>                Version of the app firmware (default: 1)
-  -c, --certs-response <certsResponse>               JSON returned by call POST /devices/{deviceid}/certificates (default: "")
-  -e, --endpoint <endpoint>                          AWS IoT MQTT endpoint (default: "")
-  -o, --device-ownership-code <deviceOwnershipCode>  PIN/ownership code of the device (default: "123456")
-  -m, --mqtt-messages-prefix <mqttMessagesPrefix>    The prefix for the MQTT for this tenant for sending and receiving device messages (default: "")
-  -v, --verbose                                      Output debug information
-  -t, --app-type <appType>                           Specifies the shadow to use. For custom shadow, pass a JSON-encoded shadow object or relative path to json file. Otherwise,
-                                                     pass "mss" or "atv2" to automatically generate a conformal shadow
-  -p, --job-execution-path <jobExecutionPath>        Specifies an unhappy job execution path for a fota update.	View the "Use an unhappy path for FOTA execution" section
-                                                     of the README for more details.																									                         
-  -h, --help                                         Output usage information
-```
-
-Use `npx @nrfcloud/device-simulator-v2 --help` to see the most recent list of options.
-
-## Caching
-The device simulator makes use of a local cache for things like device certs. This is an effort to minimize the requests made to the API. Clearing the cache is easy. 
-
-From the directory in which you ran the simulator:
-```sh
-rm -rf .ez-cache
-```
-
-Or, if actively working on the repo: 
-```js
-yarn clean
-```
-Note this is geared towards a Unix/Linux environment. Adjust commands accordingly for Windows.
-
-## Contributing
-```sh
-# install deps
+git clone git@github.com:nRFCloud/device-simulator-v2.git 
+cd device-simulator-v2
 yarn
-
-# install jq
-https://stedolan.github.io/jq/download/
-
-# modify your files
-
-# compile
-yarn build
-
-# test
-node dist/cli.js <options>
+node dist/cli.js --help
 ```
-
-### Publishing
```

Comment:
Don't we still need this section? Otherwise people who run it by `npx` won't see changes only committed to the repo. If we only want Nordicans to be able to publish, should we put this info in e.g. Confluence where outsiders can't see it?

Reply:
Yeah, I was wondering about this. I don't think we need it. No one outside of Nordic has ever contributed anyway.

Reply:
Restored